packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -32,43 +72,103 @@ export function FormatFile({ file, onDelete }: FormatFileProps) {
   }
 
   const style = imageSource ? { backgroundImage: `url(${imageSource})` } : {};
+  const imageBlockStyle = isComplete
+    ? styles.imageBlock
+    : classNames(styles.imageBlock, styles.imageBlockOverlay);
 
   return (
-    <div className={styles.formatFile}>
-      <div className={styles.imageBlock} style={style} data-testid="imageBlock">
-        {!imageSource && (
-          <div className={styles.icon}>
-            <Icon name={iconName} />
+    <>
+      {display === "file" && (
+        <div className={styles.formatFile}>
+          <div
+            className={imageBlockStyle}
+            style={style}
+            data-testid="imageBlock"
+          >
+            {!imageSource && (
+              <div className={styles.icon}>
+                <Icon name={iconName} color="greyBlue" />
```

Comment:
Reading the description of the PR it says 

> we left icon colours for non image file types to be original colours, instead of changing to grey like in the thumbnail mockups

However it looks like you set the colour of the icons in both of the code paths. Can you update the description to remove this line?